egs/wsj0-mix-var/Multi-Decoder-DPRNN/local/conf.yml

```diff
@@ -0,0 +1,41 @@
+# Filterbank config
+filterbank:
+  n_filters: 64
+  kernel_size: 8
+  stride: 4
+# Network config
+masknet:
+  n_srcs: [2, 3, 4, 5]
+  bn_chan: 128
+  hid_size: 128
+  chunk_size: 128
+  hop_size: 64
+  n_repeats: 8
+  mask_act: 'sigmoid'
+  bidirectional: true
+  dropout: 0
+  use_mulcat: false
+# Training config
+training:
+  epochs: 200
+  batch_size: 2
+  num_workers: 2
+  half_lr: yes
+  lr_decay: yes
+  early_stop: yes
+  gradient_clipping: 5
+# Optim config
+optim:
+  optimizer: adam
+  lr: 0.001
+  weight_decay: 0.00000
+# Data config
+data:
+  train_dir: "../../../dataset/{}speakers/wav8k/min/tr"
+  valid_dir: "../../../dataset/{}speakers/wav8k/min/cv"
```

Comment:
I think I said this before, but relative path in the config file doesn't seem right.

Reply:
ok. I'll change this to data/{}speakers/wav8k/min/tr. I just wrote relative paths because how it was on my system